tests/test_logger.py

```diff
@@ -103,10 +108,23 @@ def test_error_message(self):
             logger.trace("test exc_info", exc_info=ex)
 
     def test_logger_name_loglevel(self):
-        event, context = create_event(), LambdaContextMock()
-        logger = CumulusLogger('logger_test', logging.INFO)
-        logger.setMetadata(event, context)
+        logger = self.set_up_logger(logger=CumulusLogger('logger_test', logging.INFO))
         self.assertTrue(logger.logger.getEffectiveLevel() == logging.INFO)
         logger.debug("test logging level debug")
         logger.info("test logging level info")
         logger.warning("test logging level warning")
+
+    def test_simple_message_with_braces_no_args(self):
```

Comment:
I committed these new tests in 34ede6d125fa30fb808781890b0d914fc6abb80b before committing the fix; to see the failing behavior:

```
git checkout 34ede6d
nosetests tests/test_logger.py
```